### PR TITLE
CustomNoUpgrade for multi vCenter CI jobs

### PR DESCRIPTION
**Pull request by WenXinWei**
refer to https://issues.redhat.com/browse/OCPSTRAT-1576.
multi-vcenter GA in 4.18+. need to remove CustomNoUpgrade for multi vCenter CI jobs
related pr:https://github.com/openshift/release/pull/59174/files

### CODE REVIEW

**Comment by WenXinWei** [comment]
cc @jianlinliu @duanwei33 @sgaoshang  for review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml [hunk 1]
```diff
@@ -3469,19 +3463,13 @@ tests:
   cron: 14 5 7 * *
   steps:
     cluster_profile: vsphere-elastic
-    env:
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml:3472]: 4.18 looks good, can you help `tp` keyword for 4.17 jobs.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml:3472]: And can you add a 4.18 z-stream upgrade job for it.

**Comment by duanwei33** [comment]
IIUC we need to remove it from 4.19 as well.

**Comment by jianlinliu** [comment]
> IIUC we need to remove it from 4.19 as well.

4.19 ci jobs are enabled for now ?

**Comment by WenXinWei** [comment]
> > IIUC we need to remove it from 4.19 as well.
> 
> 4.19 ci jobs are enabled for now ?

I guess no, not found 4.19 job in release repo.

**Comment by duanwei33** [comment]
> > > IIUC we need to remove it from 4.19 as well.
> > 
> > 
> > 4.19 ci jobs are enabled for now ?
> 
> I guess no, not found 4.19 job in release repo.

Oh sorry, I was wrong :)

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml [hunk 2]
```diff
@@ -847,6 +847,26 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-multi-vcenter-tp-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml:850]: We do not run y-stream upgrade jobs for tp jobs.

Comment (WenXinWei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml:850]: oh, so i need upgrade from 4.18 stable to nightly right?

Comment (WenXinWei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml:850]: will close this pr as it already messed. 
will update in pr:https://github.com/openshift/release/pull/59500

Comment (WenXinWei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml:850]: cc @duanwei33

**Pull request closed without merging**